src/System.CommandLine/ParseResult.cs

```diff
@@ -15,48 +15,50 @@ namespace System.CommandLine
     public sealed class ParseResult
     {
         private readonly IReadOnlyDictionary<CliSymbol, ValueResult> valueResultDictionary = new Dictionary<CliSymbol, ValueResult>();
+        private SymbolLookupByName? symbolLookupByName = null;
 
-        private readonly CommandResult _rootCommandResult;
-// TODO: unmatched tokens, invocation, completion
-/*
+        private readonly CommandResult? _rootCommandResult;
```

Comment:
Why make this nullable? It seems like in it's assignment in the constructor we don't expect it to be null